source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst

```diff
@@ -0,0 +1,276 @@
+
+**Advantage**: You can execute the commands within your instance.
+
+**Disadvantage**: You have to transfer files between your local machine and your instance later in the tutorial.
+
+Transfer the ``concordium-client`` package from your machine via a file-sharing tool (such as `FileZilla <https://filezilla-project.org/>`_ or the ``sftp`` command) to your instance.
+
+Connect to your instance and make your package executable again as you have done previously already:
+
+.. code-block:: console
+
+   $chmod +x concordium-client
+
+Check if everything is connected correctly by displaying the best/latest block.
+
+.. code-block:: console
+
+   $./concordium-client block show --grpc-port 10001
+
+You should see some block data output.
+
+.. image:: ./images/pb_tutorial_18.png
+   :width: 100 %
+
+.. note::
+
+   Port 10001 is open by default on your testnet node to interact with it.
+
+**Option 2 (advanced users)**
+
+This option explains how you can use the ``concordium-client`` tool locally on your machine and connect remotely to your node running on the server.
+
+**Advantage**: You don't have to transfer files between your local machine and your instance later in the tutorial.
+
+**Disadvantage**: You have to use ssh with port forwarding when you run a command locally.
+
+Since you have a remote server your cloud provider usually gives you an option to ssh into it. Add the following port forwarding rule to your method to ssh into your instance in terminal A. The port 10001 on your localhost is forwarded to the port 10001 on your instance.
+
+.. code-block:: console
+
+   $ssh -NL localhost:10001:<IP-address-of-your-instance>:10001 <username>@<host>
+
+.. image:: ./images/pb_tutorial_26.png
+   :width: 100 %
+
+.. note::
+
+   Port 10001 is open by default on your testnet node to interact with it. Cloud providers often use ``ubuntu`` as the default <username> and the <IP-address-of-your-instance> as the default <host>.
+
+Go in another terminal B to the folder where you downloaded the ``concordium-client``. Check if everything is connected correctly by displaying the best/latest block.
+
+.. code-block:: console
+
+   $./concordium-client block show --grpc-port 10001
+
+You should see some block data output.
+
+.. image:: ./images/pb_tutorial_17.png
+   :width: 100 %
+
+
+Creating a mobile wallet account on testnet
+===========================================
+You are ready to create a mobile wallet account now.
+
+.. note::
+
+   Only the Concordium mobile wallet currently supports exporting of keys. Do not use the Concordium desktop wallet for this tutorial.
+
+You can download the mobile wallet package from the :ref:`downloads page<downloads-mobile-wallet-testnet>`.
+If you follow the instruction in the mobile app you are asked to name your account and to name your initial identity card. This tutorial uses ``Concordium`` for the account name and ``Account1`` for the identity card name but feel free to choose your own naming.
```

Comment:
I just realised that `Account1` is used for the *identity name*, which might be a bit confusing. It affects several of the photos, but please change it if it isn't too much hassle.

Reply:
I updated one photo and the text.